src/components/attributes-panel/attributes-stats/attribute-stats.tsx

```diff
@@ -295,15 +323,41 @@ export const AttributeStats = ({
               )}
             </>
           )}
-
-          <AttributeColorize>
-            <ColorizeTitle>{COLORIZE_BY_ATTRIBUTE}</ColorizeTitle>
-            <ToggleSwitch
-              id={"colorize-by-attribute"}
-              checked={true}
-              onChange={onColorizeByAttributeClick}
-            />
-          </AttributeColorize>
+          {statistics?.max && statistics?.avg && (
+            <>
+              <AttributeColorize>
+                <ColorizeTitle>{COLORIZE_BY_ATTRIBUTE}</ColorizeTitle>
+                <ToggleSwitch
+                  id={"colorize-by-attribute"}
+                  checked={showColorizeByAttribute}
+                  onChange={onColorizeByAttributeClick}
+                />
+              </AttributeColorize>
+              {showColorizeByAttribute && (
+                <>
+                  <FadeContainer>
+                    <Fade />
+                  </FadeContainer>
+                  <ColorizeValuesList>
+                    <ColorValueItem
+                      arrowVisibility={VISIBLE}
+                      deg={-90}
+                      yearCount={statistics.min}
+                    />
+                    <ColorValueItem
+                      arrowVisibility={NOT_VISIBLE}
+                      yearCount={Math.floor(statistics.avg)}
```

Comment:
I won't be able to visualize it with distribution avg calculated. Let's calculate it from min and max values